tests/CMakeLists.txt

```diff
@@ -406,11 +406,23 @@ if(WITH_PYTHON)
     TEST_FILES restarting/from_7.3.49/ClientMetricRestart-1.toml
     restarting/from_7.3.49/ClientMetricRestart-2.toml)
   add_fdb_test(
-    TEST_FILES restarting/from_7.3.0/ClientTransactionProfilingCorrectness-1.toml
-    restarting/from_7.3.0/ClientTransactionProfilingCorrectness-2.toml)
+    TEST_FILES restarting/from_7.3.0_until_7.3.5/ClientTransactionProfilingCorrectness-1.toml
+    restarting/from_7.3.0_until_7.3.5/ClientTransactionProfilingCorrectness-2.toml)
```

Comment:
Remove. I checked https://github.com/FoundationDB/fdb-joshua/blob/main/Dockerfile, where only 7.3.43 and 7.3.27 are listed. So we can ignore 7.3.0 to 7.3.5 here, as they are not supported.

Reply:
Ok. One thing to mention here is the 7.3 binary < 7.3.5 does not have the knob `dd_physical_shard_move_probability`